testing/unit/tp/node_account/test_transfer_frozen_to_unfrozen.py

```diff
@@ -0,0 +1,224 @@
+"""
+Provide tests for account handler apply (genesis) method implementation.
+"""
+import time
+
+import pytest
+
+from sawtooth_sdk.processor.exceptions import InvalidTransaction
+from sawtooth_sdk.protobuf.processor_pb2 import TpProcessRequest
+from sawtooth_sdk.protobuf.setting_pb2 import Setting
+from sawtooth_sdk.protobuf.transaction_pb2 import (
+    Transaction,
+    TransactionHeader,
+)
+
+from remme.protos.node_account_pb2 import (
+    NodeAccount,
+    NodeAccountMethod,
+    NodeAccountInternalTransferPayload,
+)
+from remme.protos.transaction_pb2 import TransactionPayload
+from remme.settings import SETTINGS_MINIMUM_STAKE
+from remme.settings.helper import _make_settings_key
+from remme.shared.utils import hash512
+from remme.tp.node_account import NodeAccountHandler
+from testing.conftest import create_signer
+from testing.mocks.stub import StubContext
+
+RANDOM_NODE_PUBLIC_KEY = '039d6881f0a71d05659e1f40b443684b93c7b7c504ea23ea8949ef5216a2236940'
+
+FROZEN = 600_000
+UNFROZEN = 100_000
+MINIMUM_STAKE = 250_000
+
+ACCOUNT_ADDRESS_FROM = '116829d71fa7e120c60fb392a64fd69de891a60c667d9ea9e5d9d9d617263be6c20202'
```

Comment:
`ACCOUNT_ADDRESS_FROM ` > `NODE_ACCOUNT_ADDRESS_FROM` and other constants.

https://github.com/Remmeauth/remme-core/pull/235#discussion_r260797190

Reply:
@dmytrostriletskyi, done.